deploy/charts/trust-manager/values.yaml

```diff
@@ -77,6 +77,43 @@ resources: {}
   #   cpu: 100m
   #   memory: 128Mi
 
+nodeSelector:
+  kubernetes.io/os: linux
+
+# expects input structure as per specification https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#affinity-v1-core
+# for example:
+#   affinity:
+#     nodeAffinity:
+#      requiredDuringSchedulingIgnoredDuringExecution:
+#        nodeSelectorTerms:
+#        - matchExpressions:
+#          - key: foo.bar.com/role
+#            operator: In
+#            values:
+#            - master
+affinity: {}
+
+# expects input structure as per specification https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#toleration-v1-core
+# for example:
+#   tolerations:
+#   - key: foo.bar.com/role
+#     operator: Equal
+#     value: master
+#     effect: NoSchedule
+tolerations: []
+
+# expects input structure as per specification https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.24/#topologyspreadconstraint-v1-core
+# for example:
+#   topologySpreadConstraints:
+#   - maxSkew: 2
+#     topologyKey: topology.kubernetes.io/zone
+#     whenUnsatisfiable: ScheduleAnyway
+#     labelSelector:
+#       matchLabels:
+#         app.kubernetes.io/instance: cert-manager
+#         app.kubernetes.io/component: controller
+topologySpreadConstraints: []
```

Comment:
suggestion: I appreciate you've copied the descriptions from cert-manager, which is helpful. However, those links are actually broken and I raised a [cert-manager PR](https://github.com/cert-manager/cert-manager/pull/5999) to fix that. Could we update here, too?

Also, the example for `topologySpreadConstraints` mentions cert-manager and the controller; could we update that to talk about trust-manager?

Reply:
These have been updated to v1.27 and links now work